### ISSUE TITLE
issue: yaml config with npx promptfoo eval

### ISSUE THREAD

**Issue by MentalGear** [typpo/promptfoo#23]
Trying to run examples in the lib that use yaml config file on 0.8.3 using

`OPENAI_API_KEY=[secret] npx promptfoo eval`
or
`OPENAI_API_KEY=[secret] npx promptfoo eval -c promptfooconfig.yaml`

results in
```bash
error: required option '-p, --prompts <paths...>' not specified
```

Only the `js-config` example, which uses a .js config, still works.
```bash
Loaded default config from promptfooconfig.js
```

It seems to be only an issue with the codebase though, not when using it as a package on another project, so maybe I'm just doing something wrong here.

**Comment by typpo** [comment]
I can't repro on a fresh repository.  What version are you running?  `npx promptfoo --version`

Alternatively, ensure that you are running the latest with `npx promptfoo@latest eval`

If that doesn't work, it would be good to see the contents of promptfooconfig.yaml.

**Comment by MentalGear** [comment]
Let me clarify: The intention if using `npx promptfoo` was to run the `bin`: `promptfoo` from the `package.json` for development, not the latest version from the npm registry.

Beforehand I used `npm prepare` and `npm build:watch`, but to no avail to get it to run.

I eventually used `npm local` (aka `ts-node --esm src/main.ts`), but I had to change all the explicit imports from `.js`, ( i.e. `import { evaluate } from './evaluator.js';` to `import { evaluate } from './evaluator';` ) since they are ts file in their native state.

At least this finally worked :).
